app/javascript/packages/unpolyfill-webpack-plugin/unpolyfill-webpack-plugin.js

```diff
@@ -0,0 +1,19 @@
+const { NormalModuleReplacementPlugin } = require('webpack');
+const manifest = require('./package.json');
+
+const polyfills = new RegExp(
+  `^${Object.keys(manifest.exports)
+    .filter((exportKey) => exportKey !== '.')
+    .map((exportKey) => exportKey.replace(/^\.\//, ''))
+    .join('|')}$`,
+);
```

Comment:
I wish there was a way to separate building the string from the map, so the `^` and `$` end up on the same line (to make the interpolation clearer), but couldn't think of a way without defining another variable
```suggestion
const expression = Object.keys(manifest.exports)
    .filter((exportKey) => exportKey !== '.')
    .map((exportKey) => exportKey.replace(/^\.\//, ''))
    .join('|');
const polyfills = new RegExp(`^${expression}$`);
```

Reply:
Agree! I think it might be cognitively helpful to have just an array of the polyfill module names anyways. Cleaned it up in 34f3097.